test/ssh_command_test.rb

```diff
@@ -69,7 +75,7 @@ def test_try_connection_with_gateway_connects_using_ssh_gateway
     cmd = command("10.0.0.1", "--ssh-gateway=user@gateway")
     ssh_mock = mock 'ssh_mock'
     Net::SSH::Gateway.expects(:new).with('gateway', 'user').returns(ssh_mock)
-    ssh_mock.expects(:ssh).with(cmd.host, cmd.user, cmd.connection_options.except(:gateway))
+    ssh_mock.expects(:ssh).with(cmd.host, cmd.user, cmd.connection_options.select{|o| o != :gateway})
```

Comment:
Yeah. You can definitely get it from ActiveSupport but I don't see anything requiring it in this repo so I'm not sure how it used to work. It certainly works when you run all the tests but fails if you just run this one.